hooks/use-deploy.ts

```
@@ -23,17 +23,37 @@ export function useDeploy(accessToken?: string | null): DeployContextValue {
     if (result || error || isLoading) setShowOverlay(true);
   }, [result, error, isLoading]);
 
+  // Helper to extract code blocks with filenames from LLM output
+  function parseFilesFromMessageContent(content: string): Array<{ file: string; data: string; encoding: string }> {
+    const fileBlocks: Array<{ file: string; data: string; encoding: string }> = [];
+    if (!content) return fileBlocks;
+    // Regex to match code blocks like ```tsx file="app/page.tsx" ...code...```
+    const codeBlockRegex = /```[a-zA-Z]+ file=\"([^\"]+)\"\n([\s\S]*?)```/g;
+    let match;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)